### PR TITLE
store/tikv: export LockResolver.

### PR DESCRIPTION
**Pull request by disksing**
For binlog service. Toke down by #3172 unexpectedly.

### CODE REVIEW

File: store/tikv/lock_resolver.go [hunk 1]
```diff
@@ -46,6 +48,22 @@ func newLockResolver(store *tikvStore) *LockResolver {
...
+// NewLockResolver creates a LockResolver.
+// It is exported for other services to use. For instance, binlog service needs
+// to determine a transaction's commit state.
+func NewLockResolver(etcdAddrs []string) (*LockResolver, error) {
+	pdCli, err := pd.NewClient(etcdAddrs)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	uuid := fmt.Sprintf("tikv-%v", pdCli.GetClusterID(goctx.TODO()))
+	s, err := newTikvStore(uuid, &codecPDClient{pdCli}, newRPCClient(), false)
```
Comment (zimulala) [inline store/tikv/lock_resolver.go]: Why use `newLockResolver(store)` directly?

**Comment by tiancaiamao** [comment]
LGTM

File: store/tikv/lock_resolver.go (hunk 1 above)
Comment (disksing) [inline store/tikv/lock_resolver.go]: Don't have store here, need to create one.

**Comment by zimulala** [comment]
LGTM

**Pull request merged**